web-app/js/aws-cognito/_config.js

```diff
@@ -0,0 +1,8 @@
+window._config = {
+    region: "ap-southeast-2",
+    cognito: {
+      UserPoolId: "ap-southeast-2_HKMh9Piwp",
+      ClientId: "ttspre6jds4fa5sjt979mtjrb", // nsp-cog01
+    },
```

Comment:
@nspool - should we use placeholder values for _UserPoolId_ and _ClientId_?

Reply:
How about just setting them to null?